lnrpc/lightning.yaml

```diff
@@ -145,3 +145,6 @@ http:
       delete: "/v1/macaroon/{root_key_id}"
     - selector: lnrpc.Lightning.ListPermissions
       get: "/v1/macaroon/permissions"
+    - selector: lnrpc.Lightning.CheckMacaroonPermissions
```

Comment:
I think this should be a POST method, otherwise it's going to be a bit tricky to use this via REST:

```
    - selector: lnrpc.Lightning.CheckMacaroonPermissions
      post: "/v1/macaroon/checkpermissions"
      body: "*"
```